### PR TITLE
Add backup / restore cluster tests and various small improvements / fixes

### PR DESCRIPTION
**Pull request by ynnt**

### CODE REVIEW

**Comment by ynnt** [comment]
tests are successful https://github.com/kuberlogic/kuberlogic/actions/runs/3043248440 @AvoidMe  @telepenin

**Review by telepenin — APPROVED** [review]

File: modules/dynamic-operator/Makefile [hunk 1]
```diff
@@ -105,15 +112,10 @@ run: manifests generate fmt vet ## Run a controller from your host.
...
-docker-build: export GOOS=linux
-docker-build: export GOARCH=amd64
-docker-build: export GO111MODULE=on
 docker-build: export LDFLAGS="-X main.ver=$(VERSION)"
-docker-build: export COMPOSE_LDFLAGS="-X main.ver=$(VERSION)"
 docker-build: test ## Build docker image with the manager.
-	go build -a -o bin/manager -ldflags=$(LDFLAGS) main.go
-	cd plugins/docker-compose && go build -a -o ../../bin/docker-compose-plugin -ldflags=$(COMPOSE_LDFLAGS) . && cd -
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o bin/manager -ldflags=$(LDFLAGS) main.go
+	cd plugins/docker-compose && CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o ../../bin/docker-compose-plugin -ldflags=$(COMPOSE_LDFLAGS) . && cd -
```
Comment (AvoidMe) [inline modules/dynamic-operator/Makefile:118]: I think you forget about `LDFLAGS` and `COMPOSE_LDFLAGS` here

Comment (AvoidMe) [inline modules/dynamic-operator/Makefile:118]: I think you forgot about `LDFLAGS` and `COMPOSE_LDFLAGS` here

**Review by AvoidMe — CHANGES_REQUESTED** [review]

File: modules/dynamic-operator/Makefile (hunk 1 above)
Comment (ynnt) [reply modules/dynamic-operator/Makefile]: no, they are passed as arguments to `go build`.
Still, I reverted back everything except goos and goarch that are platform specific

**Pull request merged**